bin/utils/normalizeSchema.ts

```diff
@@ -6,6 +6,7 @@ const metaProperties = [
   "$id",
   "title",
   "description",
+  "default",
```

Comment:
Why this change?
If I recall correctly, JSON schemas doesn't have this property